### ISSUE TITLE
Couchpotato

### ISSUE THREAD

**Issue by connke** [SynoCommunity/spksrc#1691]
Hello,

I couldn't see any reported issue for this so I thought I'd report it just in case.

The SynoCommunity CouchPotato version c295b344 is not allowing CP to get movies from Trakt Watch list.

**LOGS:**

    05-17 10:55:03ERROR[hpotato.core.plugins.base] Failed opening url in Trakt: https://api-
    v2launch.trakt.tv/sync/last_activities Traceback (most recent call last): HTTPError: 422 
    Client Error: Unprocessable Entity

This issue has been identified in the RuudBurger CouchPotato Server repository. See:
- https://github.com/RuudBurger/CouchPotatoServer/issues/4914

A pull request has been made 3 days ago in for that same repository. See:
- https://github.com/RuudBurger/CouchPotatoServer/pull/4957

Forgive me for my ignorance on such matters, but could we expect this issue to be resolved on an update to the Git master?

**Comment by Dr-Bean** [comment]
Couchpotato has a built-in update function, so we don't need to update the package. 
I see that the PR you mentioned has been merged into the develop branch: once develop is merged into master, CP will update itself and you should be good to go.

**Comment by Dr-Bean** [comment]
RuudBurger has just merged develop into master, so when your CouchPotato installation updates (or you force that manually), it should include the fix.
